### PR TITLE
Add support for envfrom

### PR DESCRIPTION
**Pull request by jmymy**
- [X] Chart version *not* bumped (the versions are all bumped and released at the same time)
- [X] README.md updated with any new values or changes
- [X] Updated template tests in `${CHART}/tests/*.py` 
- [ ] Updated integration tests in `${CHART}/examples/*/test/goss.yaml`

> envFrom: Configure all key-value pairs in a ConfigMap as container environment variables

### CODE REVIEW

**Comment by jmlrt** [comment]
jenkins test this please

File: metricbeat/templates/deployment.yaml [hunk 1]
```diff
@@ -96,6 +96,9 @@ spec:
 {{- if .Values.extraEnvs }}
 {{ toYaml .Values.extraEnvs | indent 8 }}
 {{- end }}
+{{- if .Values.envFrom }}
+        envFrom:
+{{ toYaml .Values.envFrom | indent 10 }}
```
Comment (jmlrt) [inline metricbeat/templates/deployment.yaml]: You forgot the ` {{- end }}` after your `{{- if ... }}`.

File: metricbeat/templates/daemonset.yaml [hunk 2]
```diff
@@ -124,6 +124,9 @@ spec:
 {{- if .Values.extraEnvs }}
 {{ toYaml .Values.extraEnvs | indent 8 }}
 {{- end }}
+{{- if .Values.envFrom }}
+        envFrom:
+{{ toYaml .Values.envFrom | indent 10 }}
```
Comment (jmlrt) [inline metricbeat/templates/daemonset.yaml]: You forgot the ` {{- end }}` after your `{{- if ... }}`.

**Comment by jmymy** [comment]
> jenkins test this please

eeek. Sorry about that. updated.

**Comment by jmlrt** [comment]
jenkins test this please

**Pull request merged**